packages/amplify_authenticator/example/integration_test/sign_in_force_new_password_test.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:amplify_api/amplify_api.dart';
+import 'package:amplify_authenticator/amplify_authenticator.dart';
+import 'package:amplify_authenticator/src/keys.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'config.dart';
+import 'pages/confirm_sign_in_page.dart';
+import 'pages/sign_in_page.dart';
+import 'pages/test_utils.dart';
+import 'utils/data_utils.dart';
+import 'utils/mock_data.dart';
+
+void main() {
+  final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  // resolves issue on iOS. See: https://github.com/flutter/flutter/issues/89651
+  binding.deferFirstFrame();
+
+  final authenticator = MaterialApp(
+    home: Authenticator(
+      child: const SizedBox.shrink(),
+    ),
+  );
+
+  group('Sign In with Force New Password flow', () {
+    final username = generatePhone();
+    final phoneNumber = username.substring(2); // without '+1'
+    final password = generatePassword();
+
+    // Background
+    setUpAll(() async {
+      // Given I'm running the example
+      // "ui/components/authenticator/sign-in-with-phone"
```

Comment:
I think if we wanted to reference JS we might make this ui/components/authenticator/sign-in-force-new-password (even if it's using the same env)

Reply:
Added a reference at the top - but this is to match the feature definition